data/json/artifact/artifact_item_types.json

```diff
@@ -365,5 +365,92 @@
     "copy-from": "club_wooden",
     "name": { "str": "ancient club" },
     "description": "A misshapen rod of gnarled, knotted wood.  The knots in the wood exhibit a strange fractal pattern.  It is the only one of its kind.\nIt may have unknown powers; try activating them."
+  },
+  {
+    "id": "art_bomorrean",
+    "type": "GENERIC",
+    "copy-from": "art_ring",
+    "name": { "str": "set of oddly interlocked rings" },
+    "description": "A set of three rings linked together. There's something about the way the rings connect to eachother that gives you a headache.  It is the only one of its kind.\nIt may have unknown powers; try activating them."
+  },
+  {
+    "id": "art_penrose",
+    "type": "GENERIC",
+    "copy-from": "art_pyramid",
+    "name": { "str": "strangle triangle" },
```

Comment:
Is this intended to be "strange" instead of "strangle"?

Reply:
```suggestion
    "name": { "str": "strange triangle" },
```